config/routes.rb

```diff
@@ -30,6 +25,13 @@
     delete 'logga_ut' => 'devise/sessions#destroy', as: :destroy_user_session
   end
 
+  # tags and exlinks
+  get 'ex_links/tags' => 'ex_links#list_tags'
+  get 'ex_links/del_unused_tags' => 'ex_links#del_unused_tags'
+  get 'ex_links/check_dead' => 'ex_links#check_dead'
+  get 'ex_links/check_expired' => 'ex_links#check_expired'
+  resources :ex_links
```

Comment:
Well the `exlinks/del_used_tags` could probably be a `delete`.
You still handle the return in controller?